src/commands/login/index.ts

```diff
@@ -93,22 +93,25 @@ fastify.get("/complete", async (request: any, reply: any) => {
 const start = async () => {
   try {
     const ports = await portastic.find({ min: 8000, max: 8999 });
-    const port =
-      process.env.NODE_ENV === "development"
-        ? 3000
-        : ports[Math.floor(Math.random() * ports.length)];
-    const localConsoleServer = getConsoleServer("local", port);
+    const port = ports[Math.floor(Math.random() * ports.length)];
 
     // request a jwt from the console ui
     fastify.get("/", async (request, reply) => {
       console.log(`Sending user to ${consoleServer} to authenticate`);
+
+      // the web dev server will be under port 3000, and api under port 3005
+      const webServer =
```

Comment:
I think we can just call getConsoleServer one time?

Reply:
getConsoleServer will automatically return 3005 but under dev mode the web served at 3000

however, we can't change the port 3005 to 3000 in the setting because the api used are from port 3005.

so I created an independent variable for that.